### PR TITLE
Dead code clean-up: Comment Repo & dependencies

### PR DESCRIPTION
**Pull request by raymondng**
Code in search component temporarily commented out until we move to eliza

### CODE REVIEW

File: kifi-backend/modules/search/test/com/keepit/search/comment/CommentIndexerTest.scala [hunk 1]
```diff
@@ -27,360 +27,361 @@ class CommentIndexerTest extends Specification with GraphTestHelper {
...
+//          hits2(0).id === publicComment2.id.get.id
+//        }
+//
+//        // find by site
+//        {
+//          val hits = searcher.search(new TermQuery(new Term(CommentFields.siteField, "keepit.com")))
+//          hits.size === 2
+//        }
+//      }
+//    }
```
Comment (andrewconner) [inline kifi-backend/modules/search/test/com/keepit/search/comment/CommentIndexerTest.scala:385]: This commented out for reference for when we plug it back in?

**Comment by andrewconner** [comment]
Nice. It may be helpful to remove all the commented out code dealing with Comment indexing in one commit, so the code is gone, but can easily be brought back by reverting that commit (when someone has time to plug it all back in to Eliza).

File: kifi-backend/modules/search/test/com/keepit/search/comment/CommentIndexerTest.scala (hunk 1 above)
Comment (raymondng) [inline kifi-backend/modules/search/test/com/keepit/search/comment/CommentIndexerTest.scala:385]: Yes -- that's only for the search component related code.

**Comment by raymondng** [comment]
Thanks. That's certainly an option.

**Comment by stkem** [comment]
lgtm

**Pull request merged**